modules/Auth/tests/auth_tests.cpp

```diff
@@ -1200,4 +1206,31 @@ TEST_F(AuthTest, test_master_pass_group_id) {
     ASSERT_FALSE(this->auth_receiver->get_authorization(1));
 }
 
+/// \brief Test TokenTimedOut is published when authorization was provided but transaction has not yet been started.
```

Comment:
```suggestion
/// \brief Test TokenTimedOut is published when authorization was provided but transaction has never been started.
```
I think "yet" is misleading here.